project/Dependencies.scala

```diff
@@ -24,6 +24,9 @@ object Dependencies {
   val akkaTestKitTyped = "com.typesafe.akka" %% "akka-actor-testkit-typed" % akkaVersion
   val akkaStreamTestKit = "com.typesafe.akka" %% "akka-stream-testkit" % akkaVersion
 
+  val akkaGrpcWebSupport = "ch.megard" %% "akka-http-cors" % "1.1.3"
```

Comment:
Wow `akka-http-cors` provides **web support** for akka-grpc? 😯 I mean, I've used this lib... but for CORS in akka-http 😅  mostly asking out of curiosity, that's test code anyway

Reply:
`akka-http-cors` is used by the `akka.grpc.scaladsl.WebHandler` in it's methods and I just needed to add this ti the classpath for tests... I do not know the details...